js/components/App.js

```diff
@@ -37,7 +39,7 @@ const App = ({
   GEN_WINDOWS.forEach((windowId, i) => {
     const Component = genWindowMap[windowId];
     windows[`genWindow${i}`] = openWindows.has(windowId) && (
-      <Component key={i} />
+      <Component key={i} mediaPlayer={media} />
```

Comment:
Ideally we would expose as little to the component as possible. I'm guessing we could get away with just passing the analyser node.

Reply:
Yea, all it should need is the analyzerNode